app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return dict(
+        line=error["line_number"],
+        column=error["column_number"],
+        message=error["text"],
+        name=error["code"],
+        source="flake8"
+    )
 
```

Comment:
You can simply write the dictionary and avoid using dict() method.

Using {}  is the more common way to define dictionaries.

Reply:
I need to change 5 times "=" -> ":", then 5 times to put in "" each key of dict to fixed this simple comment)